pom.xml

```diff
@@ -240,6 +240,7 @@
         <gib.referenceBranch>refs/remotes/origin/master</gib.referenceBranch>
         <gib.baseBranch>HEAD</gib.baseBranch>
         <gib.enabled>false</gib.enabled>
+        <gib.failOnError>false</gib.failOnError>
```

Comment:
❓ do you think it is worth adding a comment similar to the PR description "Updates default GIB behavior to perform a full build if it loses git context"? It may help others with why this was changed.

Reply:
Not sure, last time I had a discussion with some of the other developers about inline comments it sounded like they are considered a code smell. It may be considered differently in a pom, however.

Reply:
I can promise that not all DDF developers consider inline comments a code smell.